### DISCUSSION TITLE
Mountain Duck & Cryptomator on 2 PCs simultaneously?

### DISCUSSION THREAD

**Discussion by zakimohzani** [iterate-ch/cyberduck#14872] [Q&A] [answered]
Hello guys,

I tried searching for this and was struggling to find a concrete answer.

I have a laptop and a PC. I would like both of them to access a client-side encrypted cloud-drive. 

Can I have 2 copies of Mountain Duck+Cryptomator on 2 Windows machines work simultaneously on Dropbox/OneDrive (non-business account)?

Thank you

**Comment by dkocher** [comment, marked as answer]
I understand you want to have Mountain Duck installed on two computers and access the same vault stored on Dropbox or OneDrive. This setup is supported. Mountain Duck has its own fully interoperable implementation of Cryptomator [^1] and thus you do not need to have _Cryptomator for Windows_ installed.

[^1]: https://cyberduck.io/cryptomator/

**Reply by zakimohzani** [reply]
Thanks for the prompt reply. I'll give it a try now.
